core/core/misc/constants.py

```diff
@@ -25,3 +25,65 @@ class Actions(Enum):
     ADD_NOTE = 'add_note'
     DELETE_NOTE = 'delete_note'
     UPDATE_NOTE = 'update_note'
+
+
+class Constants(Enum):
+   #Command messages
+    BIRTHDAY_ADDED = "Birthday for contact added."
+    CONTACT_ADDED = "Contact added."
+    CONTACT_CHANGED = "Contact changed."
+    CONTACT_DELETED = "Contact deleted."
+    CONTACT_NOT_FOUND = "Contact not found."
+    CONTACTS_REMOVED = "Contacts were removed from AddressBook."
```

Comment:
What do you think about creating smaller classes like ContactsMessages, ValidationMessages etc.?

Reply:
It's not only about the contacts section. How about creating separate classes like CommandMessages and ValidationMessages?

Reply:
Yes, it was just like the example. You could name it as you wish, just with smaller classes, it would be easier to work with constants. I will know that, for example, ValidationMessages contains only messages related to validation and nothing more. And I assume there could be more than two new classes here.

Reply:
Great. Will fix it :)